content/en/getting_started/continuous_testing/_index.md

```diff
@@ -0,0 +1,153 @@
+
+Datadog also offers Parallelization, allowing you to execute multiple tests in your CI/CD pipelines simultaneously rather than sequentially to help speed up your building, testing, and deployment processes.
+
+## Prerequisites
+
+If you haven't already, create a [Datadog account][1].
+
+## Create a Continuous Testing test
+
+To set up a Continuous Testing test, first create a Synthetics test in Datadog. In this example, you'll create a [Browser test](https://docs.datadoghq.com/getting_started/synthetics/browser_test) on the site https://www.shopist.io, a test e-commerce web application.
+
+Browser tests simulate a user's journey through your web application beginning at your **Starting URL**. Ensuring your **Starting URL** is a resource in your staging environment will make it easier to test changes before moving them into production.
+
+### Configure your test details
+
+1. In the Datadog site, hover over **UX Monitoring** in the left hand menu and select **Continuous Testing**
+2. In the top right corner, click **New Test** > **Browser Test**.
+3. Define your browser test:
+
+    - Add the URL of the website you want to monitor into the Starting URL field. For this example, enter https://www.shopist.io 
+    - Select Advanced Options to set custom request options, certificates, authentication credentials, and more. In this example, no specific advanced option is needed.
+    - Name your test and set a Team Tag such as **team-checkout**. Tags allow you to keep your test suite organized and quickly find tests you’re interested in through the Synthetic Monitoring & Continuous Testing Explorer.
+    - Choose the browsers and devices to test.
+
+4. Continue [filling out your test details and your recording like you normally would](https://docs.datadoghq.com/getting_started/synthetics/browser_test/#create-a-browser-test).
+
+{{< img src="continuous_testing/new_browser_test.png" alt="new_browser_test" style="width:100%;" >}}
+
+
+## Run your Continuous Testing tests
+
+To improve your developer workflow, you can either choose to run your Continuous Testing test using the CLI or your IDE.
+
+### Running tests in the CLI
+
+Extend your use of Continuous Testing by using the [datadog-ci NPM package](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm). datadog-ci lets you execute commands from within your CI/CD scripts to test your application before deployment, allowing you to automate blocking and rolling back changes in case of any testing failures. Refer to the [datadog-ci Configuration page for installation and setup instructions](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm#install-the-package).
```

Comment:
```suggestion
Extend your use of Continuous Testing by using the [`datadog-ci` NPM package](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm). `datadog-ci` lets you execute commands from within your CI/CD scripts to test your application before deployment, allowing you to automate blocking and rolling back changes in case of any testing failures. For installation and setup instructions, see the [Configuration documentation](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm#install-the-package).
```